### ISSUE TITLE
** URGENT - Copyrighted Material ** 

### ISSUE THREAD

**Issue by RicardoAmplify** [brisingre/asenoise#1]
Greetings,

We noticed that your project includes copyrighted material made by our company. Although we really appreciate the use of Amplify Shader Editor and Bloom, public distribution of our products is not permitted.

Honest mistake, we've come across many users that were not aware of the issue. We would appreciate it if you removed it or made it private.

Please confirm via email (ricardo@amplify.pt) that you will take action.

Thanks!

**Comment by brisingre** [comment]
Fair enough. I can't really make it private, github makes you pay monthly
fees for that, but I can add the amplify folders to the git ignore.

**Comment by brisingre** [comment]
I have made the decision to make this repository private, with a new host. 

I am getting rid of the old one now. Sorry it took me so long, was a bit of trouble to decide on a free private host. 

I'm working on migrating my other unity repositories now. This one will be deleted as soon as I post this comment.
